how-to/how-to-use-virtio-fs-with-kata.md

```diff
@@ -39,7 +39,7 @@ This will place the Kata release artifacts in `/opt/kata`, and update Docker's c
 
 Once installed, start a new container, utilizing NEMU + `virtiofs`:
 ```bash
-$ docker run --runtime=kata-nemu -it busybox
+$ docker run --runtime=kata-qemu -it busybox
```

Comment:
Thanks @Bryan1269. `kata-nemu` is deprecated and this needs to be updated. But the runtime to be used here is `kata-qemu-virtiofs` instead of `kata-qemu`, as this runtime makes use of qemu with virtio-fs support.